immuni_common/models/marshmallow/validators.py

```diff
@@ -218,6 +218,9 @@ def __call__(self, value: List[TemporaryExposureKey]) -> None:
         #   here is done on purpose to make things simpler in case of changes in validation.
         #   We would like to keep field validation more flexible as not to spread strict validation
         #   logic in many places.
+        if len(value) == 0:
+            return
+
```

Comment:
I'd move these lines before the NOTE. In fact, the NOTE is strictly related to the `if any(tek.rolling_period != 144 for tek in value):` validation.

Reply:
Sure thing.